server/lib/autostart.coffee

```diff
@@ -102,7 +103,8 @@ start = (appli, callback) ->
                         if not appli.permissions
                             password = appli.password
                             delete appli.password
-                        clientDS = new Client "http://#{dsHost}:#{dsPort}"
+                        dsUrl = "#{dsSchema}://#{dsHost}:#{dsPort}"
+                        clientDS = new Client dsUrl
```

Comment:
Because here the port can changed: https://github.com/kosssi/cozy-controller/blob/add_cozy_url_sdk/server/lib/autostart.coffee#L229
I don't know why.